site_root/input/resources/extensions/StructureDefinition-us-core-ethnicity.json

```diff
@@ -0,0 +1,2133 @@
+  },
+  "url": "http://hl7.org/fhir/us/core/StructureDefinition/us-core-ethnicity",
+  "version": "3.1.0",
+  "name": "USCoreEthnicityExtension",
+  "title": "US Core Ethnicity Extension",
+  "status": "active",
+  "date": "2019-05-21T14:00:00+10:00",
+  "publisher": "HL7 US Realm Steering Committee",
+  "contact": [
+    {
+      "telecom": [
+        {
+          "system": "url",
+          "value": "http://www.healthit.gov"
+        }
+      ]
+    }
+  ],
+  "description": "Concepts classifying the person into a named category of humans sharing common history, traits, geographical origin or nationality.  The ethnicity codes used to represent these concepts are based upon the [CDC ethnicity and Ethnicity Code Set Version 1.0](http://www.cdc.gov/phin/resources/vocabulary/index.html) which includes over 900 concepts for representing race and ethnicity of which 43 reference ethnicity.  The ethnicity concepts are grouped by and pre-mapped to the 2 OMB ethnicity categories: - Hispanic or Latino - Not Hispanic or Latino.",
+  "jurisdiction": [
+    {
+      "coding": [
+        {
+          "system": "urn:iso:std:iso:3166",
+          "code": "US",
+          "display": "United States of America"
+        }
+      ]
+    }
+  ],
+  "purpose": "Complies with 2015 Edition Common Clinical Data Set for patient race.",
+  "fhirVersion": "4.0.1",
+  "mapping": [
+    {
+      "identity": "rim",
+      "uri": "http://hl7.org/v3",
+      "name": "RIM Mapping"
+    }
+  ],
+  "kind": "complex-type",
+  "abstract": false,
+  "context": [
+    {
+      "type": "element",
+      "expression": "Patient"
+    }
+  ],
+  "type": "Extension",
+  "baseDefinition": "http://hl7.org/fhir/StructureDefinition/Extension",
+  "derivation": "constraint",
+  "snapshot": {
```

Comment:
I think you can get rid of the `snapshot` since this resource has the `differential` defined and that should suffice

Reply:
https://www.hl7.org/fhir/us/core/StructureDefinition-us-core-ethnicity.json.html